front/app/containers/Admin/pagesAndMenu/containers/CustomPages/Edit/Content/TopInfoSection/index.tsx

```diff
@@ -14,10 +14,20 @@ const TopInfoSection = () => {
     return null;
   }
 
+  const updateCustomPageAndEnableSection = (customPageId, data) => {
+    return updateCustomPage(customPageId, {
+      ...data,
+      top_info_section_enabled: true,
+    });
+  };
+
```

Comment:
* Could you type function parameters? We're trying to get to strict mode, so let's not introduce more any instances.
* I'd rename this to enable instead of update if the function only sets it to true.

Reply:
The new data is also passed in on line 19, meaning the user can both update the page with new data and subsequently enable it

Reply:
Ok, went to quickly over it then. Ignore that comment